### ISSUE TITLE
Fix broken test due to #397

### ISSUE THREAD

**Issue by amitaibu** [RESTful-Drupal/restful#402]
#397

**Comment by amitaibu** [comment]
Update: Issue is with `Sub requests` test

**Comment by amitaibu** [comment]
hmm, seems to pass on my local (php 5.6). I'll re-run the test

**Comment by amitaibu** [comment]
@mateu-aguilo-bosch are you able to reproduce locally?

**Comment by e0ipso** [comment]
This may be due to changes to any of the dependencies. Maybe EntityValidator?

**Comment by amitaibu** [comment]
I'll check.

**Comment by amitaibu** [comment]
Indeed, once EntityValidator was updated I get the error. @RoySegall can you please have a look

**Comment by RoySegall** [comment]
Checking.

**Comment by RoySegall** [comment]
I checked out to entity validator 1.1 release and the tests still failed even entity validator tests are passing. I turn on my debugger and put brake point at the first line of `entityValidate` and the debugger did not stopped there. After comment out [this](https://github.com/RESTful-Drupal/restful/blob/7.x-1.x/plugins/restful/RestfulEntityBase.php#L667-L669) part the debugger stopped at the first line of `entityValidate` but the tests failed.

I talked about this with @amitaibu and he said this can't be the reason since all the tests going through this point and passing. I'll keep checking.

**Comment by amitaibu** [comment]
I'll comment out the failing test until entity-validator would be fixed.
